### ISSUE TITLE
v1.2 missing on packagist

### ISSUE THREAD

**Issue by robfrancken** [madnest/madzipper#28]
Hi there,

First of all thanks for providing this package :)

I'm trying to install v1.2 in my Laravel 8 application, but I'm getting a composer error:

```sh
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires madnest/madzipper 1.2, found madnest/madzipper[dev-master, v0.0.1, v1.0.0, ..., v1.3.x-dev] but it does not match the constraint.
```

**Comment by theimerj** [comment]
Hello @robfrancken, you are right, i will look into that shortly.

**Comment by robfrancken** [comment]
Thank you.

I was able to install `1.2.x-dev`, however I'm getting the same error as per #21

**Comment by theimerj** [comment]
Hmm, I neglected this package for some time now, let me see what I can do.

**Comment by theimerj** [comment]
Just published `v1.2.1` and `v1.3.1` which hopefully fix #21. Tests currently do not cover it.

**Comment by robfrancken** [comment]
Awesome, thank you. That seems to have fixed it
